package.json

```diff
@@ -39,7 +39,8 @@
     "ember-load-initializers": "^1.0.0",
     "ember-qunit-assert-helpers": "^0.2.1",
     "ember-resolver": "^4.0.0",
-    "ember-source": "http://builds.emberjs.com/beta/shas/08bc55d20f76f874b6a7ceefd4007dde1aa7c9dc.tgz",
```

Comment:
Yarn is complaining about this so I updated it

Reply:
Seems fine (we want to use a tagged release anyways), though I am curious what yarns issue was?

Reply:
![screen shot 2018-01-17 at 5 07 27 pm](https://user-images.githubusercontent.com/166909/35075645-27b98c9a-fba9-11e7-8174-64c175e90de9.png)
@rwjblue This is the error